.build.yml

```diff
@@ -153,8 +152,8 @@ before_scripts:
   - make install
 
 build_scripts:
-  - ./autogen.sh
-  - scan-build $CHECKERS ./configure --prefix=/usr --enable-pulse
+  - ./autogen.sh --enable-compile-warnings=maximum
```

Comment:
Not really an issue, but it is Ok to use `./autogen.sh` without `warn_flags`, because `make` won't executed after `autogen.sh`. See build logs.
The important part is `scan-build`.

Reply:
done